lib/web/apiserver.go

```diff
@@ -534,10 +577,26 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 	// get all SAML connectors
 	samlConnectors, err := h.cfg.ProxyClient.GetSAMLConnectors(false)
 	if err != nil {
-		log.Infof("Cannot retrieve SAML connectors: %v", err)
+		log.Errorf("Cannot retrieve SAML connectors: %v.", err)
```

Comment:
will it log an error if SAML connectors are not configured? (in which case this is not an error)

Reply:
Nope, it's not an error.